include/z64cutscene.h

```diff
@@ -86,13 +86,13 @@ typedef struct {
 } CsCmdTextbox; // size = 0xC
 
 typedef enum {
-    /*      0 */ CS_TEXTBOX_TYPE_DEFAULT,
-    /*      1 */ CS_TEXTBOX_TYPE_1,
-    /*      2 */ CS_TEXTBOX_TYPE_LEARN_SONG,
-    /*      3 */ CS_TEXTBOX_TYPE_3,
-    /*      4 */ CS_TEXTBOX_TYPE_BOSSES_REMAINS,
-    /*      5 */ CS_TEXTBOX_TYPE_ALL_NORMAL_MASKS,
-    /* 0xFFFF */ CS_TEXTBOX_TYPE_NONE = 0xFFFF
+    /* -1 */ CS_TEXTBOX_TYPE_NONE = -1,
```

Comment:
Enums are ints, so I don't think this will necessarily be equivalent.

Reply:
Seems to match fine?

Reply:
Hmm, seems like this is only used in one place anyway.